codice/ddf
# [2.26.x] DDF-6554 Handles epoch dates in tika transformer

#### What does this PR do?

#### Who is reviewing it? 
@derekwilhelm 
@glenhein 
@vinamartin 

#### Select relevant component teams: 
@codice/data 

#### Ask 2 committers to review/merge the PR and tag them here.
@derekwilhelm 
@glenhein 

#### How should this be tested?
Ingest mp4 testMp4ExcelEpochDates.mp4 and testMp4UnixEpochDates and ensure that no epoch dates appear in the metacard.  
Ingest other tika-supported datatypes and ensure the same thing

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #6554 

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
